Nodejs/Setup/NodejsToolsInstaller/NodejsToolsInstaller.wxs

```diff
@@ -153,11 +157,25 @@
         <CustomAction Id="VWDExpressSetup" Property="VWDEXPRESS_PATH" ExeCommand="/setup" Execute="deferred" Return="check" Impersonate="no" />
         <CustomAction Id="VWDExpressSetup_Rollback" Property="VWDEXPRESS_PATH" ExeCommand="/setup" Execute="rollback" Return="check" Impersonate="no" />
 
+      <Binary Id="InstallerTelemetryAction.CA.dll" SourceFile="..\..\..\BuildOutput\$(var.Configuration)$(var.VSTargetVersion)\Setup\Microsoft.NodejsTools.InstallerTelemetryAction.CA.dll" />
+      
+      <!-- Custom Action for installer telemetry-->
+      <CustomAction Id="LogStart" BinaryKey="InstallerTelemetryAction.CA.dll" DllEntry="RecordInstallStartTime"/>
+      <CustomAction Id="LogSuccess" BinaryKey="InstallerTelemetryAction.CA.dll" DllEntry="LogInstallSuccessResult"/>
+      <CustomAction Id="LogFailure" BinaryKey="InstallerTelemetryAction.CA.dll" DllEntry="LogInstallErrorResult"/>
+      <CustomAction Id="LogCancel" BinaryKey="InstallerTelemetryAction.CA.dll" DllEntry="LogInstallCancelResult"/>
```

Comment:
You will also need to add these dlls to dev scripts - add them to generate.js (before VsLogger for the same reason as above), run the script, and check in the result 

See https://github.com/Microsoft/nodejstools/tree/master/Nodejs/Prerequisites